docs/articles/documentation/using-testcafe/programming-interface/BrowserConnection.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title: BrowserConnection Class
+permalink: /documentation/using-testcafe/programming-interface/BrowserConnection/
+---
+# BrowserConnection Class
+
+Conection to a remote browser.
+
+Created by the [testCafe.createBrowserConnection](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/#createBrowserConnection) function.
+
+#### Example
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+
+var remoteConnection = testcafe.createBrowserConnection();
+
+// Outputs remoteConnection.url to visit it from the remote browser.
+console.log(remoteConnection.url);
+remoteConnection.on('ready', () => {
+    await testCafe
+        .createRunner()
+        .browsers(remoteConnection)
+        .run();
+});
+```
+
+## Properties
+
+### <a class="anchor" name="url"></a>url → String
+
+A URL that should be visited from a remote browser to connect.
+
+## Events
+
+### <a class="anchor" name="ready"></a>ready
+
+Fires when a remote browser has been connected.
```

Comment:
I guess it requires a mention that you can omit waiting for the `ready` in your code. Runner will automatically wait for 30 seconds for all it's connections to be ready before firing the error